src/main/scala/com/ing/wbaa/gargoyle/sts/data/UserInfo.scala

```diff
@@ -1,5 +1,9 @@
 package com.ing.wbaa.gargoyle.sts.data
 
+case class UserName(value: String) extends AnyVal
```

Comment:
Reason for the case class is that this info is used in the `UserInfo` and `KeycloakUserInfo` classes. Since this is supposed to be the same user, a case class enforces this. Same for UserGroup case class

Reply:
ok